datasets/wikiqa.py

```diff
@@ -10,7 +10,7 @@
 from datasets.idf_utils import get_pairwise_word_to_doc_freq, get_pairwise_overlap_features
 
 
-class WikiQA(CastorPairDataset):
+class WIKIQA(CastorPairDataset):
```

Comment:
I changed this to lower case and it's now changed back haha..

Reply:
oh...Sorry. It is due to my messy local code. I will learn from it. Thanks!